test/ComputationCoordinatorPrototypeSmoke.java

```diff
@@ -2,6 +2,7 @@
 import api2.ComputationCoordinator;
 import api2.ComputationCoordinatorEmpty;
 import api2.ComputeEngine;
+import api2.ComputeEngineEmpty;
```

Comment:
A lot of the issues I've faced with the project so far have mostly been from confusion on what to import. I think once I get a better handle on where that can cause issues, it may click for me better